src/pptrees/prefix_graph.py

```diff
@@ -530,6 +533,13 @@ def hdl(self,out=None,full_flat=False):
         if out is not None:
             with open(out,'w') as f:
                 print(hdl,file=f)
+            # Copy mapping file from package to local directory
+            with importlib.resources.path("pptrees","mappings") as pkg_map_dir:
+                pkg_map_file = pkg_map_dir / (mapping+'_map.v')
+            local_map_file = os.path.join(os.path.dirname(os.path.abspath(out)))
```

Comment:
it might be easier to use https://docs.python.org/3/library/pathlib.html here rather than `os.path` since `importlib.resources.path` return you a `pathlib.Path` object: https://docs.python.org/3/library/importlib.html#importlib.resources.path